src/pydata_sphinx_theme/toctree.py

```diff
@@ -32,12 +32,48 @@ def add_inline_math(node: Node) -> str:
     )
 
 
+def get_unrendered_local_toctree(
```

Comment:
this is all copy-pasted from `index_toctree` at the end of this file (which now calls this helper).  Needed to move to the beginning of the file so that `get_sidebar_toctree_length` can also call it.